### PR TITLE
[Feature] add seatunnel monitor

### PR DESCRIPTION
**Pull request by zhangshenghang**
## What's changed?

- reslove #2455 

image

image

## Checklist

- [ ]  I have read the [Contributing Guide](https://hertzbeat.apache.org/docs/community/code_style_and_quality_guide)
- [ ]  I have written the necessary doc or comment.
- [ ]  I have added the necessary unit tests and all cases have passed.

## Add or update API

- [ ] I have added the necessary [e2e tests](https://github.com/apache/hertzbeat/tree/master/e2e) and all cases have passed.

### CODE REVIEW

**Review by zqr10159 — APPROVED** [review]

File: home/docs/help/seatunnel.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
+---
+id: seatunnel
+title: Monitoring： Seatunnel
+sidebar_label: Seatunnel
+keywords: [ Open Source Monitoring System, Monitor Seatunnel ]
```
Comment (Hisoka-X) [inline home/docs/help/seatunnel.md:5]:
```suggestion
title: Monitoring： SeaTunnel
sidebar_label: SeaTunnel
keywords: [ Open Source Monitoring System, Monitor SeaTunnel ]
```

File: home/docs/help/seatunnel.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+---
+
+> Collect monitoring metrics for Seatunnel.
```
Comment (Hisoka-X) [inline home/docs/help/seatunnel.md:8]:
```suggestion
> Collect monitoring metrics for SeaTunnel.
```

File: home/docs/help/seatunnel.md [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+
+### Configuration Parameters
+
+| Parameter Name    | Parameter Help Description                                                                                      |
+|-------------------|-----------------------------------------------------------------------------------------------------------------|
+| Target Host       | The monitored endpoint's IPV4, IPV6, or domain name. Note ⚠️ no protocol header (e.g., https://, http://).      |
+| Task Name         | The name that identifies this monitoring task, which needs to be unique.                                        |
+| Port              | The monitoring port opened by Seatunnel, default value: 5801.                                                   |
```
Comment (Hisoka-X) [inline home/docs/help/seatunnel.md:16]:
```suggestion
| Port              | The monitoring port opened by SeaTunnel, default value: 5801.                                                   |
```

**Review by Hisoka-X — COMMENTED** [review]
Thanks @zhangshenghang bring SeaTunnel to hertzbeat!

**Review by Hisoka-X — APPROVED** [review]

**Review by Aias00 — APPROVED** [review]
LGTM

**Review by yuluo-yx — APPROVED** [review]
lgtm

**Pull request merged**